shpc/main/modules/base.py

```diff
@@ -106,24 +106,26 @@ def uninstall(self, name, force=False):
         self.container.delete(name)
 
         if container_dir != module_dir:
-            self._uninstall(container_dir, "$container_base/%s" % name, force)
+            force = force or self._uninstall(container_dir, "$container_base/%s" % name, force)
```

Comment:
Any reason not to just do:
```suggestion
            force = self._uninstall(container_dir, "$container_base/%s" % name, force)
```

Reply:
Thinking more about this - if you want just one force, why not move the ask (confirm action) outside of those two functions to just ask once?

Reply:
> Any reason not to just do:

Yes, that'd work too :)

> Thinking more about this - if you want just one force, why not move the ask (confirm action) outside of those two functions to just ask once?

My reasoning was to let the user choose whether they want to remove:
 - nothing
 - the module only
 - the module and the container

And thinking that once the symlinks would be added, the four options possible scenario would be:
- nothing
- the symlink only
- the symlink and the module
- the symlink, the module, and the container

i.e. not allowing dangling symlinks or modules, as could happen if we ask about each component separately